tests/service/scheduler/test_scheduler.py

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright 2021 Swiss Data Science Center (SDSC)
+# A partnership between École Polytechnique Fédérale de Lausanne (EPFL) and
+# Eidgenössische Technische Hochschule Zürich (ETHZ).
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Renku service tests for scheduler management."""
+from rq import Queue
+
+
+def test_enqueue_jobs(with_scheduler):
+    """Enqueue jobs at a interval."""
+    queues = Queue.all()
+    assert queues
+
+    assert 2 == len(queues)
+    for q in queues:
+        assert 1 == q.count
```

Comment:
Could we add a project and file with expired ttl, and then check that they get cleaned up after this?

Reply:
This is testing only the queuing up the jobs, not the job execution itself. The job execution itself is tested in `test_cache.py`